cmd/talosctl/cmd/talos/processes.go

```diff
@@ -190,7 +190,7 @@ func processesOutput(ctx context.Context, c *client.Client) (output string, err
 
 	s := []string{}
 
-	s = append(s, "NODE | PID | STATE | THREADS | CPU-TIME | VIRTMEM | RESMEM | COMMAND")
+	s = append(s, "NODE | PID | STATE | THREADS | CPU-TIME | VIRTMEM | RESMEM | LABEL | COMMAND")
```

Comment:
Should we add a flag to only show these when wanted as typically users shouldn't need SELinux labels?

Reply:
I'd say fine to have a label always